### DISCUSSION TITLE
At least a few words about monitoring

### DISCUSSION THREAD

**Discussion by orlovds** [Keyfactor/ejbca-ce#719] [Q&A] [answered]
Hi. 
Please tell me, is there information about the parameters of monitoring and integration with prometeus somewhere?

**Comment by primetomas** [comment, marked as answer]
Find some documentation about monitoring in the documentation. While not specifically about prometeus it should give you some good insights.

https://docs.keyfactor.com/ejbca/latest/monitoring-and-healthcheck
https://docs.keyfactor.com/ejbca/latest/monitor-ejbca-host-using-monit
https://docs.keyfactor.com/ejbca/latest/integrating-ejbca-with-azure-application-insights
https://docs.keyfactor.com/ejbca/latest/3key-dashboarding-monitoring-and-reporting-add-on
https://docs.keyfactor.com/ejbca/latest/logging

**Comment by orlovds** [comment]
@primetomas 
I see, thank you very much!
